g4f/Provider/hf_space/Microsoft_Phi_4_Multimodal.py

```diff
@@ -15,7 +15,7 @@
 from .DeepseekAI_JanusPro7b import get_zerogpu_token
 from .raise_for_status import raise_for_status
 
-class Microsoft_Phi_4(AsyncGeneratorProvider, ProviderModelMixin):
+class Microsoft_Phi_4_Multimodal(AsyncGeneratorProvider, ProviderModelMixin):
```

Comment:
Why you change the provider name?
And if you change it, you need to update the examples in media,md